packages/eas-cli/src/commandUtils/flags.ts

```diff
@@ -10,6 +20,42 @@ export const EasNonInteractiveAndJsonFlags = {
   }),
 };
 
+export const EASEnvironmentFlag = {
+  environment: Flags.enum({
+    description: 'Environment to create the secret in',
+    parse: upperCaseAsync,
+    options: mapToLowercase([
+      EnvironmentVariableEnvironment.Development,
+      EnvironmentVariableEnvironment.Preview,
+      EnvironmentVariableEnvironment.Production,
+    ]),
```

Comment:
```suggestion
    options: [
      EnvironmentVariableEnvironment.Development,
      EnvironmentVariableEnvironment.Preview,
      EnvironmentVariableEnvironment.Production,
    ].map(e => e.toLowerCase()),
```

nit: do we really need the `mapToLowerCase` function?

Reply:
We don't need it, but we use it in two places and this looks IMO nicer. But it's just a preference.